_data/scala-compiler-team.yml

```
@@ -0,0 +1,104 @@
+# - name: Core Team
+#   description: Knows everything
+#   image: /resources/img/frontpage/scala-spiral.png
+#   city: Lausanne, Switzerland
+#   email: scala.center@epfl.ch
+#   twitter: scala_lang
+#   github: scala
+#   linkedin: https://www.linkedin.com/in/janedoe/
+#   mastodon: https://fosstodon.org/@janedoe
+#   meetup: https://www.meetup.com/your/url/here
+
+- name: Martin Odersky
+  description: Language Creator
+  github: odersky
+
+- name: Piotr Chabelski
+  description: Product Manager
+  github: Gedochao
+
+- name: Tomasz Godzik
+  description: VirtusLab Compiler Team Leader
+  github: tgodzik
+
+- name: Oliver Bračevac
+  description: LAMP Compiler Team Leader, Specialised Engineer (CC)
+  github: bracevac
+
+- name: Dale Wijnand
+  description: Compiler Engineer, Specialised Engineer (Matching)
+  github: dwijnand
+
+- name: Matt Bovel
+  description: Compiler Engineer
+  github: mbovel
+
+- name: Eugene Flesselle
+  description: Compiler Engineer
+  github: EugeneFlesselle
+
+- name: Hamza Remmal
+  description: Infrastructure Manager
```

NIT typo in role name
```suggestion
  description: Infrastructure Maintainer
```